cosmwasm/contracts/pyth/src/msg.rs

```diff
@@ -12,17 +12,26 @@ type HumanAddr = String;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct InstantiateMsg {
-    pub wormhole_contract:  HumanAddr,
-    pub pyth_emitter:       Binary,
-    pub pyth_emitter_chain: u16,
+    pub wormhole_contract:          HumanAddr,
+    pub pyth_emitter:               Binary,
+    pub pyth_emitter_chain:         u16,
+    pub governance_emitter:         Binary,
+    pub governance_emitter_chain:   u16,
+    pub governance_sequence_number: u64,
+    pub chain_id:                   u16,
+    pub valid_time_period_secs:     u16,
+
+    pub fee:       u128,
+    pub fee_denom: String,
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum ExecuteMsg {
-    SubmitVaa { data: Binary },
+    UpdatePriceFeeds { data: Binary },
```

Comment:
Not related to this PR, but would be nice if we add UpdatePriceFeedsIfNecessary in another PR too.